tools/benchmark/src/main/java/com/datadog/benchmark/internal/BenchmarkSpanToSpanEventMapper.kt

```diff
@@ -28,19 +28,25 @@ internal class BenchmarkSpanToSpanEventMapper {
             duration = durationNanos,
             start = spanData.startEpochNanos,
             error = 0, // error is not needed in benchmark
-            meta = resolveMeta(),
+            meta = resolveMeta(spanData),
             metrics = resolveMetrics()
         )
     }
 
-    private fun resolveMeta(): SpanEvent.Meta {
-        // TODO: RUM-5985 Fill SpanEvent with useful meta data
+    private fun resolveMeta(spanData: SpanData): SpanEvent.Meta {
+        val map = mutableMapOf<String, String>()
+
+        spanData.attributes.forEach { t, u ->
```

Comment:
what `t` and `u` stand for?

Reply:
it's `AttributeKey` and `Any`(value), updated with meaningful names